build/webpack/font-awesome.js

```diff
@@ -0,0 +1,12 @@
+addCSS('https://use.fontawesome.com/releases/v5.8.1/css/all.css');
```

Comment:
Doesn't this mean this will never work for users without internet connection?
Not sure this is what we want. Is this what we do in webviews?

Reply:
> This is much simpler and then we can just remove our shipping of it altogether.

Shouldn't we identify the problem & fix it.
E.g. i WSL users don't have network connection, hence this will not work even for basic widgets thats built in.
I would prefer we identify the cause and fix it.

Reply:
Also we're shipping 	4.7.0 and using 5.8.1 here, not sure this is correct. Also would be incorrect in the licensing docs..

Reply:
Old approach fails because we can't get a webviewURI at the time of the preload.

Reply:
preloads are on the kernel, not the webview? Maybe there's a way to do a preload on the webview?

Reply:
And I'm not sure about WSL. We can test it out. I can do a curl of say http://www.google.com from a WSL bash script.